go/install/install.go

```diff
@@ -259,3 +258,15 @@ func IsInUse(mountDir string, log Log) bool {
 	}
 	return false
 }
+
+// TerminateApp will stop the Keybase (UI) app
+func TerminateApp(context Context, log Log) error {
+	appExecName := "Keybase"
+	logf := logger.NewLoggerf(log)
+	log.Info("Stopping Keybase app")
+	appPIDs := process.TerminateAll(process.NewMatcher(appExecName, process.ExecutableEqual, logf), 5*time.Second, logf)
```

Comment:
The matcher here is for `ExecutableEqual` which means it will only match processes with Keybase executable name (exactly) and not any variation.

I kind of like doing executable name instead of path in case a person did something weird and put it in a weird place (it would still terminate). (We also have a check on startup, that it's installed in the right place.)